easybuild/easyconfigs/a/AmberTools/AmberTools-17-foss-2018a.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'EB_Amber'
+
+name = 'AmberTools'
+version = '17'
+
+homepage = 'http://ambermd.org/'
+description = """AmberTools consists of several independently developed packages that work well by themselves,
+ and with Amber itself. The suite can also be used to carry out complete molecular dynamics simulations,
+ with either explicit water or generalized Born solvent models."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+toolchainopts = {'usempi': True}
+
+# download requires registration, see http://ambermd.org/AmberTools17-get.html
+sources = ['AmberTools%(version)s.tar.bz2']
+patches = ['Amber-16_fix-hardcoding.patch']
```

Comment:
Should probably add all the AT-17 patches from Amber-16 in a/Amber/

That should take care of a whole bunch of problems